urllib3/response.py

```diff
@@ -387,48 +386,67 @@ def readinto(self, b):
             b[:len(temp)] = temp
             return len(temp)
 
-    def read_chunked(self, amt=None):
+    def _get_next_chunk(self):
```

Comment:
The name of this method is confusing, because it doesn't get the chunk, it gets the chunk length.